packages/Takeover/src/Takeover.js

```diff
@@ -84,25 +86,29 @@ export default function Takeover(props) {
     return headerExtracted.props.children;
   }
 
-  return React.cloneElement(takeoverOverlay, overlayProps, state => (
-    <sc.Takeover
-      state={state}
-      {...moreProps}
-      aria-label={getAriaLabel()}
-      aria-modal
-      data-pka-anchor="takeover"
-      role="dialog"
-      zIndex={zIndex}
-    >
-      {headerExtracted && <sc.Header {...headerExtracted.props} onClose={onClose} />}
-      {contentExtracted && (
-        <sc.Content role="region" tabIndex="0">
-          {contentExtracted}
-        </sc.Content>
+  return (
+    <OriginalOverlay {...overlayProps}>
+      {state => (
```

Comment:
why is this turn into a render function? I'm not understanding the context.

Reply:
it was a render function before as well, https://github.com/acl-services/paprika/pull/934/files/97c7748ee1ea3f36898a5fddc36ab2519afbff15#diff-7f8e871aef4f5eab982513c0ec898d0387c02987deb2af97b5d961413ab0fea4L87 look at the children parameter was a function. This is a requirement from `@paprika/overlay` component